cmd/metrics.go

```diff
@@ -497,10 +497,11 @@ func storageMetricsPrometheus(ch chan<- prometheus.Metric) {
 		return
 	}
 
-	// Fetch disk space info, ignore errors
-	storageInfo, _ := objLayer.StorageInfo(GlobalContext, true)
+	server := getLocalServerProperty(globalEndpoints, &http.Request{
+		Host: GetLocalPeer(globalEndpoints),
+	})
 
-	onlineDisks, offlineDisks := getOnlineOfflineDisksStats(storageInfo.Disks)
+	onlineDisks, offlineDisks := getOnlineOfflineDisksStats(server.Disks)
```

Comment:
This is a breaking change, but I like that for prometheus since users generally want to have information per server

Reply:
It's not breaking it's still the value represented per server.